kolibri/plugins/management/assets/src/vue/data-page/index.vue

```diff
@@ -53,6 +48,28 @@
   const coreApp = require('kolibri');
 
   module.exports = {
+    $trNameSpace: 'manageData',
+    $trs: {
+      // Headings/subHeadings
+      pageHeading: 'Export Usage Data',
+      pageSubHeading: 'Download CSV (comma-separated value) files containing information about' +
```

Comment:
I think this is the most general way of approaching this, and as @indirectlylit mentioned, it is no longer a performance concern.